docs/pages/docs/index.mdx

```diff
@@ -9,22 +9,22 @@ import Features from "../../components/Features";
 
 <Features page="docs" detailed={false} />
 
-Turborepo is a blazing fast build system for JavaScript/TypeScript _monorepos_: codebases containing multiple
-projects, often using multiple frameworks, in a single unified
+Turborepo is a build system for JavaScript/TypeScript _monorepos_: codebases containing multiple
+projects, often using multiple frameworks, in a single, unified
 code repository.
 
-Monorepos have many advantages, but they require appropriate tooling in order to scale. Turborepo provides this tooling and more. It leverages advanced build system ideas and techniques to speed up development, but with a fraction of the configuration and complexity. Instead of wasting countless days worrying about _how_ you're shipping, Turborepo lets you focus on _what_ you're shipping by abstracting all the annoying configuration, scripts, and tooling on your behalf.
+Monorepos have many advantages, but they require appropriate tooling in order to scale. Turborepo provides this tooling and more. It leverages advanced build system ideas and techniques to speed up development, but with a fraction of the configuration and complexity. Turborepo abstracts away all the annoying configuration, scripts, and tooling into `turbo.json`, so you can focus on _what_ you're shipping, instead of worrying about _how_ it'll get shipped.
```

Comment:
```suggestion
Monorepos have many advantages, but they require appropriate tooling to scale. Turborepo provides this tooling and more. It leverages advanced build system ideas and techniques to speed up development, but with a fraction of the configuration and complexity. Turborepo abstracts all the annoying configurations, scripts, and tooling into `turbo.json`, so you can focus on _what_ you're shipping instead of worrying about _how_ it'll get shipped.
```

Reply:
I like the edit other than "annoying configurations" (plural), I think it should remain singular. Will accept the suggestion and push up another edit later!